themes/classic/templates/catalog/manufacturer.tpl

```diff
@@ -4,7 +4,7 @@
   <section id="main">
 
     {block name='manufacturer_header'}
-      <h1>{l s='List of products by manufacturer %name%' d='Shop.Theme.Catalog' sprintf=['%name%' => $manufacturer.name]}</h1>
+      <h1>{l s='List of products by brand %name%' d='Shop.Theme.Catalog' sprintf=['%name%' => $brand.name]}</h1>
```

Comment:
You need to leave the `$brand` variable. It's not updated into the controller yet.